### ISSUE TITLE
sharing ship/freighter seeds

### ISSUE THREAD

**Issue by Albirew** [goatfungus/NMSSaveEditor#285]
I believe either this issue or the wiki should be used for users to share ships / freighters skins with screenshot. 

eg with my freighter:
home seed (dunno if usefull): 0x23700FBDA4680
model seed: 0xBC8F8D94D5FFD0DA
screenshot: 
![20200719224438_1](https://user-images.githubusercontent.com/2805161/87887019-9faf0380-ca21-11ea-951f-6d8fea43ec45.jpg)

**Comment by melhiore** [comment]
I don't think this is the right place for this. Reddit coordinate exchange maybe?? NMseeds will be happy to host this information but this Github is for something different...

**Comment by Albirew** [comment]
i believe the project wiki should be good enough since it's actually not used
